dracut/anaconda-lib.sh

```diff
@@ -382,7 +387,7 @@ run_kickstart() {
     # net: re-run online hooks
     if [ "$do_net" ]; then
         # If NetworkManager is used in initramfs
-        if [ -e "$hookdir"/cmdline/*-nm-config.sh ]; then
+        if ls -U "$hookdir"/cmdline/*-nm-config.sh >/dev/null 2>&1 ; then
```

Comment:
I wonder what  `-U` is, I can't find it in the man page on my Fedora...

Reply:
On f32 I have it? It's "do not sort before output". Technically it saves performance in cases where you have many files (millions). Perhaps it is not necessary here.

Reply:
Hmm, I see it now... I wonder why I missed it before. Yeah, could stay this way.